doc/packages/build-support.md

```diff
@@ -0,0 +1,97 @@
+# Build Support {#sec-build-support}
+
+## `pkgs.substitute` {#pkgs-substitute}
+
+`pkgs.substitute` is a wrapper around [the `substitute` function](#fun-substitute) provided by the
+stdenv. It replaces strings in `src` as specified by the `substitutions` argument.
+
+In a build script, the line:
```

Comment:
Please use `:::{.example}` blocks for all examples. This makes them more discoverable in the examples section, and easier to discern in the text body.